src/ReconNess/Services/TargetService.cs

```diff
@@ -231,6 +231,57 @@ public async Task<Target> ExportTargetAsync(Expression<Func<Target, bool>> crite
                         .FirstOrDefaultAsync(cancellationToken);
         }
 
+        public async Task<Target> ExportTargetWithRootDomainsOnlyAsync(Expression<Func<Target, bool>> criteria, CancellationToken cancellationToken = default)
```

Comment:
We dont need a new method I think. We need to return all the data including subdomains/services/directories, notes, etc. 

https://github.com/reconness/reconness/blob/8e9378f85e7f4cdca14a7560a8879b0b655cc141/src/ReconNess/Services/TargetService.cs#L139